src/hub/action_request.ts

```diff
@@ -355,6 +355,14 @@ export class ActionRequest {
       } else {
         return sanitizeFilename(`looker_file_${Date.now()}.${this.attachment.fileExtension}`)
       }
+    } else {
```

Comment:
Let's add some tests to https://github.com/looker/actions/blob/master/test/test_action_request.ts for the suggested filename. Lots of conditionals and now a util function in here now.